packages/react-native-popup-menu-android/package.json

```diff
@@ -21,9 +21,15 @@
     "@react-native/codegen": "0.75.0-main"
   },
   "peerDependencies": {
+    "@types/react": "^18.0.0",
```

Comment:
Could we align everything in the repo to ^18.2.6?

It looks like we have two versions in use already. But raw `18.0.0` is incompatible with RN types because of a deprecation that happened during 18 typings releases.

Reply:
done